packages/ERTP/src/types.js

```diff
@@ -97,6 +97,19 @@
  * to set subtraction.
  */
 
+/**
+ * @typedef {Object} DisplayInfo
+ * @property {number} decimalPlaces
+ *   Tells the display software how many decimal places to move the
+ *   decimal over to the left, or in other words, which position corresponds to whole
+ *   numbers. We require fungible digital assets to be represented in
+ *   integers, in the smallest unit (i.e. USD might be represented in mill,
+ *   a thousandth of a dollar. In that case, `decimalPlaces` would be 3.)
+ *   For non-fungible digital assets, this should be left as undefined.
+ *   The decimalPlaces property should be used for *display purposes only*. Any
+ *   other use is an anti-pattern.
```

Comment:
Nice caution! 

Btw, JS Internationalization has a bunch of stuff about formatting currency. Those formatters are also only about rendering things into strings, not the underlying representation. I can imagine we might eventually want to grow this options bag in that direction. But not yet ;)

Reply:
Agreed!